tailwind.config.js

```diff
@@ -129,6 +129,75 @@ module.exports = {
         800: violetDark.violet6,
         900: violet.violet12,
       },
+      primary: {
```

Comment:
Let's retain all the colors under a new name. We can eliminate the old colors once we transition every component to the new design.